modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -2936,6 +2936,70 @@ definition {
 		}
 	}
 
+	@description = "LPS-164948 - Verify that the Formula Builder is not searchable"
```

Comment:
The verbiage is inconsistent here. The description refers to the Formula Builder but the BDD steps refer to the Formula Field. We should be consistent throughout the test so that it's clear what the purpose of the test is.